### PR TITLE
(SERVER-2060) Prep for 1.0.0 release

### PR DESCRIPTION
**Pull request by justinstoller**

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -4,20 +4,5 @@
   "issues": "https://tickets.puppetlabs.com/browse/SERVER",
   "internal_list": "https://groups.google.com/a/puppet.com/forum/?hl=en#!forum/discuss-puppetserver-maintainers",
   "people": [
```
Comment (haus) [inline MAINTAINERS]: want to just delete this file?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
+### 1.0.0 - 2018-01-09
+
+* (SERVER-1780) Insert hashCode into profiler file name
```
Comment (haus) [inline CHANGELOG.md]: hmmmm?

Comment (justinstoller) [reply CHANGELOG.md]: derp

File: MAINTAINERS (hunk 1 above)
Comment (justinstoller) [reply MAINTAINERS]: I kinda just wanted to add you as schadenfreude

Comment (haus) [reply MAINTAINERS]: i know where you are

**Comment by justinstoller** [comment]
updated

File: MAINTAINERS (hunk 1 above)
Comment (justinstoller) [reply MAINTAINERS]: No you don't. I'm not at my desk

Comment (justinstoller) [reply MAINTAINERS]: Damnit

**Pull request merged**